app/models/task_item.rb

```diff
@@ -6,35 +6,19 @@ class TaskItem < ApplicationRecord
   scope :completed, -> { where.not(completed_at: nil) }
   scope :incomplete, -> { where(completed_at: nil) }
 
-  validates :name, presence: true
```

Comment:
![code review comment 2](https://github.com/solid-process/solid-rails-app/assets/2651240/7627ff4e-de73-4dd2-ad46-7be141ec9485)

<img width="603" alt="image" src="https://github.com/solid-process/solid-rails-app/assets/2651240/5f9b5e7b-cd8b-4218-9799-6cd7049387cf">

Reply:
Based on the excerpt from [the comment I made](https://github.com/solid-process/solid-rails-app/pull/10#discussion_r1590328273) in your other contribution

> ...removing presence validation doesn't make sense, as it is used both to create understandable feedback for the API users and to avoid unnecessary I/O in the database (string checking is absurdly more efficient than allowing the query to be made).

Beyond this, I would like to ask you to bring contributions without removing any feature. As I also said in the other comment:

> This project aims to use the common Rails features in the vanilla-Rails branch and then show the equivalence when using the solid-process gem.

Once again, thank you for your time and collaboration.